script/Ciel/Day 2/02_01_CIEL02_1_3.txt

```diff
@@ -146,27 +156,30 @@ I follow her example and sit down next to her at a slight distance.
 [sha:9602ac95d14eaaf72ea437d07aa21e52d4b6270e]{
 -- Page 59, Offset 3758.
 -- 一年前に姿を消したカニクリームコロッケロール？」
-The crab cream croquette rolls that vanished a year ago?"
+Did they bring back the crab cream croquette rolls that vanished a year ago?"
 }
 [sha:8056f758d46d34217e8c9f28adf4b12c7d4cd9ec]{
 -- Page 59, Offset 3759.
 -- ……昔、そういう素敵なメニューがあったのだ。
-...There used to be a great menu like that.
+...They really did have a delicacy like that on the menu, back in the day.
 }
 [sha:1b6c01eace4726573d1e8ea32de0f93746c095aa]{
 -- Page 59, Offset 3760.
 -- あまりの高コストで一週間で生産中止になったが、今でも復活を待つ生徒は少なくない。
-It was so expensive that it was discontinued within a week, but many students are still waiting for it to return.
+They were so expensive that they were discontinued after the first week,
+ but a lot of students are still holding out hope they'll make a comeback.
 }
 [sha:8e6d4c3ca6f594aaaa9756a196be53ca28b903fa]{
 -- Page 60, Offset 3761.
 -- 「ふふふ、それはですね……」
-"Fufufu, well you see..."
+"Hehehe, well you see..."
 }
 [sha:a1057f2e5e10e2d6399b74f232029b5c62385e8e]{
 -- Page 60, Offset 3762.
 -- 先輩は嬉しさでとろけるように笑って紙袋を膝に置くと、中から四つの総菜パンを取りだした。
-Senpai smiles as if melting with joy, places the paper bag on her lap, and takes out four side dishes from inside.
+// 総菜パン doesn't have a great analog, but 'stuffed bread' used elsewhere in script
+Senpai is smiling fit to burst as she places the paper bag on her lap.
```

Comment:
comma after smiling

Reply:
(smiling fit to burst) is a phrase, not sure I'd break it up with a comma

Reply:
alright

Reply:
not an american phrase. In fact, I've never heard it before as far as i can remember
https://www.ldoceonline.com/dictionary/fit-to-burst

Reply:
:soubrit: 

Positively beaming? Or is that too UK as well?

Reply:
positively beaming is fine